Sources/RemoteMessaging/Mappers/JsonToRemoteMessageModelMapper.swift

```diff
@@ -56,20 +57,25 @@ private enum AttributesKey: String, CaseIterable {
         case .favorites: return FavoritesMatchingAttribute(jsonMatchingAttribute: jsonMatchingAttribute)
         case .appTheme: return AppThemeMatchingAttribute(jsonMatchingAttribute: jsonMatchingAttribute)
         case .daysSinceInstalled: return DaysSinceInstalledMatchingAttribute(jsonMatchingAttribute: jsonMatchingAttribute)
-        case .isNetPWaitlistUser: return IsNetPWaitlistUserMatchingAttribute(jsonMatchingAttribute: jsonMatchingAttribute)
         case .daysSinceNetPEnabled: return DaysSinceNetPEnabledMatchingAttribute(jsonMatchingAttribute: jsonMatchingAttribute)
+        case .pproEligible: return IsPrivacyProEligibleUserMatchingAttribute(jsonMatchingAttribute: jsonMatchingAttribute)
+        case .pproSubscriber: return IsPrivacyProSubscriberUserMatchingAttribute(jsonMatchingAttribute: jsonMatchingAttribute)
         }
     }
 }
 // swiftlint:enable cyclomatic_complexity
 
 struct JsonToRemoteMessageModelMapper {
 
-    static func maps(jsonRemoteMessages: [RemoteMessageResponse.JsonRemoteMessage]) -> [RemoteMessageModel] {
+    static func maps(jsonRemoteMessages: [RemoteMessageResponse.JsonRemoteMessage],
+                     surveyActionMapper: RemoteMessagingSurveyActionMapping) -> [RemoteMessageModel] {
         var remoteMessages: [RemoteMessageModel] = []
         jsonRemoteMessages.forEach { message in
             var remoteMessage = RemoteMessageModel(id: message.id,
```

Comment:
@samsymons I stumbled across a pre-existing bug while testing - would you mind adding a fix as part of this PR? If `mapToContent` returns `nil` the message is still appended to the list of remoteMessages to display. The message won't display (since it's content is nil) but it also means that if there is a second valid message after it, that second message won't ever be shown to the user


```
            guard let content = mapToContent( content: message.content, surveyActionMapper: surveyActionMapper) else {
                return
            }
            var remoteMessage = RemoteMessageModel(id: message.id,
                                              content: content,
                                              matchingRules: message.matchingRules ?? [],
                                              exclusionRules: message.exclusionRules ?? [])
```

Reply:
Fix applied and tests updated - good spotting, thanks!